.github/workflows/test.yml

```diff
@@ -107,7 +107,7 @@ jobs:
       - run: sam init --name sam-app --runtime nodejs14.x --dependency-manager npm --app-template hello-world
```

Comment:
We should probably update this if we are upgrading NodeJS versions. 14.x is on it's way out for Lambda: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

Reply:
correct